src/signature.js

```diff
@@ -51,6 +51,15 @@ export class Signature {
   armor(config = defaultConfig) {
     return armor(enums.armor.signature, this.write(), undefined, undefined, undefined, config);
   }
+
+  /**
+   * 
+   * @returns Returns an array of KeyIDs of all of the issuers who created this signature
+   * @returns {Array<KeyID>} The IDs of the issuers
+   */
+  getIssuerIDs() {
+    return this.packets.map(packet => packet.issuerKeyID)
```

Comment:
Technically speaking, there could be a marker packet here, as well, which wouldn't have an issuer key ID. So it might be good to still do something like `Message#getSigningKeyIDs` does, i.e. get the signature packets here explicitly.